topics/fair/tutorials/ro-crate-in-galaxy/tutorial.md

```diff
@@ -81,25 +83,27 @@ Let’s run the workflow and export the RO-Crate.
 >    https://zenodo.org/record/1319069/files/iris.csv
 >    ```
 >
->    Tip: Importing via links:
->
->    - Copy the link location
->    - Open the Galaxy Upload Manager (galaxy-upload on the top-right of the tool panel)
->    - Select Paste/Fetch Data
->    - Paste the link(s) into the text field
->    - Press Start
->    - Close the window
+>     > <tip-title> Importing via links </tip-title>
+>     >
+>     >   - Copy the link location
+>     >   - Open the Galaxy Upload Manager (galaxy-upload on the top-right of the tool panel)
+>     >   - Select Paste/Fetch Data
+>     >   - Paste the link(s) into the text field
+>     >   - Press Start
+>     >   - Close the window
+>     {: .tip}
 >
 > 2. Run **GTN Training: Galaxy 101 For Everyone** workflow using the following parameters:
 >    - *"Send results to a new history"*: `No`
 >    - *"1: Iris Dataset""*: the `iris.csv` file we just uploaded
 >
->    Tip: Running a workflow:
->    - Click on Workflow on the top menu bar of Galaxy. You will see a list of all your workflows.
->    - Click on the workflow-run (Run workflow) button next to your workflow
->    - Configure the workflow as needed
->    - Click the Run Workflow button at the top-right of the screen
->    - You may have to refresh your history to see the queued jobs
+>     > <tip-title> Running a workflow </tip-title>
+>     >   - Click on Workflow on the top menu bar of Galaxy. You will see a list of all your workflows.
+>     >   - Click on the {% icon workflow-run %} (Run workflow) button into your workflow box
+>     >   - Configure the workflow as needed
+>     >   - Click the {% icon workflow-run %} Run Workflow button at the top-right of the screen
+>     >   - You may have to refresh your history to see the queued jobs
+>     {: .tip}
```

Comment:
for this tip box you could just use {% snippet faqs/galaxy/workflows_run.md %} (and feel free to edit the snippet itself if needed)